src/streamlit_analytics2/main.py

```diff
@@ -428,6 +572,7 @@ def stop_tracking(
     load_from_json: Optional[Union[str, Path]] = None,
     streamlit_secrets_firestore_key: Optional[str] = None,
     firestore_project_name: Optional[str] = None,
+    session_id: Optional[str] = None,
```

Comment:
Thank you for making it optional. I have no issue merging this, pending a discussion about session privacy in the other comment